src/README.md

```diff
@@ -1,24 +1,18 @@
-# Getting Set Up
-
-For this tutorial you will need a testnet account.  If you do not have one go [here](https://wallet.testnet.near.org), using the email option is the easiest method for ensuring you can recover your account.
```

Comment:
The original idea was to have them create subaccounts for each of the contracts.

Reply:
I originally kept a modified version of this line in here, but ditched it when I saw that we already do that at the point that they actually need a testnet account in Chapter 1.5: RAEN Admin